.evergreen/config.yml

```diff
@@ -809,22 +822,19 @@ functions:
           ECS_SRC_DIR=$AUTH_AWS_DIR/src
 
           # fix issue with `TestData` in SERVER-46340
-          sed -i '1s+^+TestData = {};\n+' $AUTH_AWS_DIR/lib/ecs_hosted_test.js
+          # sed -i '1s+^+TestData = {};\n+' $AUTH_AWS_DIR/lib/ecs_hosted_test.js
```

Comment:
Tests are passing with this code commented out - should this be removed entirely?

Reply:
I've removed.